src/handlers/dispersion.ts

```diff
@@ -30,3 +30,13 @@ export const getUniswapV2Tokens = async (
     return res?.data?.tokens;
   }
 };
+
+export const getDPIBalance = async () => {
```

Comment:
would be nice to have a type definition of the format of the result since this is a TS file